src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.ts

```diff
@@ -126,44 +120,34 @@ export class RbdFormComponent implements OnInit {
         desc: this.i18n('Object map (requires exclusive-lock)'),
         requires: 'exclusive-lock',
         allowEnable: true,
-        allowDisable: true
+        allowDisable: true,
+        initDisabled: true
       },
       journaling: {
         desc: this.i18n('Journaling (requires exclusive-lock)'),
         requires: 'exclusive-lock',
         allowEnable: true,
-        allowDisable: true
+        allowDisable: true,
+        initDisabled: true
       },
       'fast-diff': {
         desc: this.i18n('Fast diff (requires object-map)'),
```

Comment:
I think we need to update the information regarding the relation between `object-map` and `fast-diff`. 
ATM its strange to read `fast-diff` requires `object-map`, but when you disable it, it disables both without saying why.

Not sure if the following suggestion is enough, but its a start:
```suggestion
        desc: this.i18n('Fast diff (interlocked with object-map)'),
```